e2e_samples/dataset_versioning/README.md

```diff
@@ -1,79 +1,67 @@
 
 ### How to use the sample
 
-1. Setup Infra
-    1. Provision infra with Terraform - [detailed steps](./infra/README.md)
-    1. Add user permission (Credential passthrough) - [details steps](./databricks/README.md)
-        1. Add your READ and EXECUTE permission in ACL (Storage account -> containers -> container(datalake) -> Manage ACL)
-        1. Open up [Microsoft Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) and right click on container to add propagate access control list.
-        > Propagate access control list cannot be found: Try updating azure storage explorer to the latest version.
-        > You might already have acl permissions if you provision resources with your credentials (e.g. az login)
-    1. Create table and install stored procedure - [detailed steps](./datafactory/config/README.md)
-    1. Deploy application logic (ARM templates) for Data Factory - [detailed steps](./datafactory/README.md)
-1. Functional test
+1. Deploy sample infrastructure, ADF pipelines and AzureSQL objects.
+    1. Provision infrastructure with Terraform. - [detailed steps](./infra/README.md)
+    1. Create AzureSQL Database tables and stored procedure. - [detailed steps](./sql/ddl/README.md)
+    1. Deploy Data Factory Pipelines. - [detailed steps](./datafactory/README.md)
+2. Running the sample
     1. Load data into data source (Azure SQL Database)
-        1. Use Python script to load "LendingClub issued Loans" data - [detailed steps](./insert_sql/README.md)
-    1. Run Azure Data Factory pipeline to load Delta Lake
-        1. Go to provisioned Azure Data Factory, then click [Author & Monitor] button to open development portal.
-        1. Click pencil button at left pane, then select [DeltaCopyPipeline].
-        1. Cick [Debug] button to run pipeline.
-    1. Use Databricks to query Delta versions
-        1. Go to Azure Databricks and create a cluster. Remember to tick the "enable credential passthrough" check under the advanced settings if you plan to use the credential passthrough feature. Make sure you have access to Deltalake either with a RBAC role or Access Control List. [detailed steps](./databricks/README.md)
-        1. Create a notebook in Azure Databricks. Select your cluster and execute these scripts below:
-
-            ```python
-            # For credential passthrough
-            configs = {
-                "fs.azure.account.auth.type": "CustomAccessToken",
-                "fs.azure.account.custom.token.provider.class": spark.conf.get("spark.databricks.passthrough.adls.gen2.tokenProviderClassName")
-            }
+        1. Use Python script to load "LendingClub issued Loans" data - [detailed steps](./sql/data_generator/README.md)
+    2. Run Azure Data Factory pipeline to load Delta Lake
+        1. Go to provisioned Azure Data Factory, then click **Author & Monitor** button to open development portal.
```

Comment:
Sorry, this is very minor.
You don't need a full stop in bullet points especially some of them already dont have a full stop

Reply:
Good point. I typically add periods if the bullet point is a full sentence but will skip it if its just a phrase.